uat/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -1,20 +1,19 @@
 @LogManager
 Feature: Greengrass V2 LogManager
 
-  As a customer, I want to selectively upload my logs and metrics to AWS Cloudwatch to save cost.
+    As a customer, I want to selectively upload my logs and metrics to AWS Cloudwatch to save cost.
 
-  Background:
-    Given my device is registered as a Thing
-    And my device is running Greengrass
-    And 5 temporary rotated log files for component aws.greengrass.Nucleus
-    And 5 temporary rotated log files for component aws.greengrass.StreamManager
```

Comment:
Shall we still include the plugin component? Although we don`t really use it, but people may question if the LM is compatible with other plugin component.

Reply:
Basically trying to copy pasta, not trying to change the tests yet